docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-256.adoc

```diff
@@ -0,0 +1,42 @@
+== Rails Master Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|90f9b054-003f-4e00-85b2-fadec8b66dc5
+
+|Checkov Check ID
+|CKV_SECRET_256
+
+|Severity
+|MEDIUM
+
+|Subtype
+|build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+The Rails Master Key is a highly sensitive credential used in Ruby on Rails applications for encrypting sensitive data, such as session cookies and database credentials. If your Rails Master Key is exposed, attackers could potentially decrypt sensitive information, hijack user sessions, or gain unauthorized access to your application and its data.
+
+=== Fix - Buildtime
+
+*Rails Master Key*
+
```

Comment:
To mitigate the risks associated with exposed Rails Master Keys, follow these steps: